src/common/utils.ts

```diff
@@ -108,13 +128,20 @@ export function formatError(e: HookError | any): string {
 	}
 
 	let errorMessage = e.message;
-	const furtherInfo = (e as HookError).errors && (e as HookError).errors!.map((error: any) => {
-		if (typeof error === 'string') {
-			return error;
-		} else {
-			return error.message;
-		}
-	}).join(', ');
+	let furtherInfo: string | undefined
+	if (e.message === 'Validation Failed' && hasFieldErrors(e)) {
+		furtherInfo = e.errors.map(error => {
+			return `Value ${error.value} cannot be set for field ${error.field} (code: ${error.code})`;
```

Comment:
I would surround ${error.value} in quotes to make it clear this is a user provided value